lib/locreator.js

```diff
@@ -34,22 +33,11 @@ module.exports.addGenericMethods = function generics(nemo) {
     return drivex.visible(_by(nemo, splitLocator(locatorString)));
   };
   nemo.view._wait = function (locatorString, timeout) {
-    function waitReturnElement() {
-      return drivex.waitForElement(_by(nemo, splitLocator(locatorString)), timeout || 5000);
-    }
-
-    var wep = new nemo.wd.WebElementPromise(nemo.driver, waitReturnElement());
-    return wep;
+    return drivex.waitForElementPromise(_by(nemo, splitLocator(locatorString)), timeout);
```

Comment:
I know this PR doesn't touch this, however I feel that we need to add `msg` argument as part of these `_wait` methods. So optionally users can provide error message when elements timeout.